week8/trivia/index.js

```diff
@@ -0,0 +1,24 @@
+let correct = document.querySelector('.correct');
+correct.addEventListener('click', function() {
+    correct.style.backgroundColor = 'green';
+    document.querySelector('#feedback1').innerHTML = 'Correct!';
+});
+
+let incorrects = document.querySelectorAll('.incorrect');
+for (let i = 0; i < incorrects.length; i++) {
```

Comment:
you can iterate over that array of stuffs with the forEach loop.

https://www.w3schools.com/jsref/jsref_foreach.asp